scripts/erd/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#
+# ERD: Entity Relationship Diagrams
+#
+# SEE:
+#
+# - https://erdantic.drivendata.org/stable/
+#
+
+ARG PYTHON_VERSION="3.9.12"
+FROM python:${PYTHON_VERSION}-slim-buster as base
```

Comment:
sometimes I wonder if the non-slim version without installing 10 tools is not better than the slim version + 10 tools in term of size and build time.

Reply:
yep. we could explore that. @mguidon showed me some tools we could use in the new HB registry. It might in addition help us decide more secure base images